test/e2e/setup/setup.go

```diff
@@ -228,6 +199,25 @@ func deployOperator() error {
 	return nil
 }
 
+func deployCertManager(config TestConfig) error {
+	const helmChartName = "cert-manager"
+	if err := helm.Uninstall(helmChartName, config.CertManagerNamespace); err != nil {
+		return err
```

Comment:
[q] do you think it's worth wrapping the error with context saying whether or not the error occurred during uninstall vs/install?

Reply:
good point, done